tox.ini

```diff
@@ -86,8 +86,6 @@ deps =
     juju
     pytest
     git+https://github.com/charmed-kubernetes/pytest-operator.git
-allowlist_externals =
-    git
```

Comment:
git is still used in the commands. Wouldn't removing this trigger a red tox warning?

Reply:
yes, you're correct. must have been a cut and paste error. my bad